typings/webgme.d.ts

```diff
@@ -1574,7 +1681,7 @@ declare namespace GmeClasses {
          * @return a dictionary where every the key of every entry is an absolute path of a set owner node. 
          * The value of each entry is an array with the set names in which the node can be found as a member.
          */
-        isMemberOf(node: Core.Node): Core.DataObject;
```

Comment:
What was Core.DataObject again?

Reply:
It's used wrongly throughout, but I think the intention was that it should represent the raw data that maps to a node. In other words the json objects stored in the monodb.

Reply:
*It was